tests/test_e2e_10_mef_eline.py

```diff
@@ -1999,3 +1999,26 @@ def test_210_bulk_update_evc(self):
         assert "test" in data[evc_1_id]["metadata"]
         assert "test" in data[evc_2_id]["metadata"]
 
+    def test_250_run_bulk_sdntraces_any_vlan(self):
```

Comment:
Since the objective is to test "/traces" endpoint with an `any` or `untagged` UNI EVC, let's exercise the "/traces" endpoint after the EVC is created. Can you move these two tests to test_e2e_40_sdntrace.py? 

Also you can [reuse the `self.create_evc` method there](https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/tests/test_e2e_40_sdntrace.py#L30). These tests should also replace `test_075_run_sdntrace_any_vlan` and `test_070_run_sdntrace_untagged_vlan` and not use flow_manager directly anymore. Let's keep it as close to the use case as possible as we've discussed on https://github.com/kytos-ng/mef_eline/pull/303.